nautobot/circuits/models.py

```diff
@@ -240,7 +240,10 @@ class Meta:
         unique_together = ["circuit", "term_side"]
 
     def __str__(self):
-        return "Side {}".format(self.get_term_side_display())
+        return "{} - Side {}".format(self.circuit.cid, self.get_term_side_display())
```

Comment:
Is this actually used? I think `get_term_side_display()` doesn't actually exist as a function any more - I don't see where it's defined, at least.

Reply:
Interesting - I could not find it either and yet the code works.  I have replaced it with `self.term_side`.

Reply:
Ah, my mistake - this is a built in Django magic method https://docs.djangoproject.com/en/3.2/ref/models/instances/#django.db.models.Model.get_FOO_display - so it should be correct to use. Sorry for the noise.

Reply:
Note that https://github.com/nautobot/nautobot/pull/1144/files#diff-794c53468afb4c48643b23cf3768a7d4850c2f7d39cad3ec38beee45c4dd23a1R310 is going to be changing this method again in `next` - would be good to make sure these two are compatible, as here you're adding the circuit CID to the representation, while #1144 is going in the direction of adding the site/provider-network to the representation instead.